### PR TITLE
[util/make_new_dif] fix checklist_template_file path

### PR DESCRIPTION
**Pull request by itay-sadov**
update checklist_template_file  path to new template file location

### CODE REVIEW

File: util/make_new_dif.py [hunk 1]
```diff
@@ -189,7 +189,7 @@ def main():
             out_files))
 
         if "checklist" in args.only:
-            checklist_template_file = REPO_TOP / "doc/project/sw_checklist.md.tpl"
+            checklist_template_file = REPO_TOP / "doc/project_governance/checklist/sw_checklist.md.tpl"
```
Comment (engdoreis) [inline util/make_new_dif.py:192]: CI failed because this line as more than 100 characters, could you break it?

**Review by engdoreis — COMMENTED** [review]
Thanks @itay-sadov for the fix. If you search for `sw_checklist.md.tpl` you'll find other locations where this path should be fixed, would mind fixing that as well?

**Review by marnovandermaas — APPROVED** [review]
Yes, I think this path is correct. Have you been able to test the tool after the change?

**Review by engdoreis — APPROVED** [review]

**Comment by marnovandermaas** [comment]
Rerunning failed CI because the error did not seem to be related to this PR.

**Comment by itay-sadov** [comment]
> @itay-sadov, could you rebase and push again, I suspect this may fix the CI error.

will do

**Comment by engdoreis** [comment]
@itay-sadov the PR https://github.com/lowRISC/opentitan/issues/18932 was merged today to fix the CI errors we are seeing here. Could you rebase again the catch the fix?

**Review by HU90m — APPROVED** [review]

**Pull request merged**